client/me/account/index.jsx

```diff
@@ -53,7 +55,7 @@ const user = _user();
  */
 let debug = new Debug( 'calypso:me:account' );
 
-module.exports = React.createClass( {
+const Account = React.createClass( {
```

Comment:
In these types of components where we end up creating a purely-UI component and then exporting a data-wrapped version of it, I really like the idea of going ahead and exporting the purely-UI part of it. That kind of thing makes testing way easier.

```js
// splineReticulator.js
import React from 'react';
import Spline from 'client/lib/spline';

export const SplineReticulator = ( { reticulate } ) => <Spline onClick={ reticulate } />;

export default connectToSplineHub( SplineReticulator );
```

In this case, all I've done differently is add `export` in front of my component, but in the test file, I can do this…

```js
// test.js

const fakeReticulator = done => () => done();

it( 'Should finish the test', done => {
	<SplineReticulator reticulate={ fakeReticulator( done ) } />;
} );
```

Here our test has no dependencies on the store or any store - just the actual data/methods that the component needs.